### PR TITLE
F200 initial support

### PR DESCRIPTION
**Pull request by reaganlo**
Fixes Issue: #15 #65 

Changes proposed in this pull request:
- Publish Depth, Color and IR data for F200 camera
- Enable dynamic reconfigure for supported F200 camera options
- Automated unit tests for F200 camera

### CODE REVIEW

File: realsense_camera/cfg/f200_params.cfg [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+PACKAGE="realsense_camera"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+#             Name                               Type      Level       Description                     Default    Min     Max
+gen.add("enable_depth",                          bool_t,   0,          "Enable Depth",                 True)
```
Comment (rjingar) [inline realsense_camera/cfg/f200_params.cfg:10]: Did you try dynamic enable/disable color stream for F200? If its working, we should add that too.

**Comment by mdhorn** [comment]
My previous comments were addressed.
We may look at refactoring the launch and test files with the addition of the next camera.

Approved. 👍

**Comment by rjingar** [comment]
Approved.

**Comment by reaganlo** [comment]
Thanks for your review comments. FYI, I have updated the README with the F200 related information.

**Pull request merged**